src/components/RangeSlider/README.md

```diff
@@ -225,6 +227,116 @@ class RangeSliderExample extends React.Component {
 }
 ```
 
+### Dual thumb range slider
+
+Use a dual thumb range slider when merchants need to select a range of values.
+
+```jsx
+const initialValue = [900, 1000];
+
+class RangeSliderExample extends React.Component {
+  state = {
+    intermediateTextFieldValue: initialValue,
+    value: initialValue,
+  };
+
+  handleRangeSliderChange = (value) => {
+    this.setState({value, intermediateTextFieldValue: value});
+  };
+
+  handleLowerTextFieldChange = (value) => {
+    const upperValue = this.state.value[1];
+    this.setState({intermediateTextFieldValue: [parseInt(value), upperValue]});
+  };
+
+  handleUpperTextFieldChange = (value) => {
+    const lowerValue = this.state.value[0];
+    this.setState({intermediateTextFieldValue: [lowerValue, parseInt(value)]});
+  };
+
+  handleLowerTextFieldBlur = () => {
+    const upperValue = this.state.value[1];
+    const value = this.state.intermediateTextFieldValue[0];
+
+    this.setState({value: [parseInt(value), upperValue]});
+  };
+
+  handleUpperTextFieldBlur = () => {
+    const lowerValue = this.state.value[0];
+    const value = this.state.intermediateTextFieldValue[1];
+
+    this.setState({value: [lowerValue, parseInt(value)]});
+  };
+
+  handleEnterKeyPress = (event) => {
+    const newValue = this.state.intermediateTextFieldValue;
+    const oldValue = this.state.value;
+
+    if (event.keyCode === Key.Enter && newValue !== oldValue) {
+      this.setState({value: newValue});
+    }
+  };
+
+  render() {
+    const {value, intermediateTextFieldValue} = this.state;
+    const prefix = '$';
+    const min = 0;
+    const max = 2000;
+    const step = 10;
+
+    const lowerTextFieldValue =
+      intermediateTextFieldValue[0] === value[0]
+        ? value[0]
+        : intermediateTextFieldValue[0];
+    const upperTextFieldValue =
+      intermediateTextFieldValue[1] === value[1]
+        ? value[1]
+        : intermediateTextFieldValue[1];
+
+    return (
+      <Card sectioned>
+        <div style={{marginTop: '20px'}} onKeyDown={this.handleEnterKeyPress}>
+          <RangeSlider
+            label="Money spent is between"
```

Comment:
For the dual-thumb version, the `<label>` is technically invalid since there isn't actually an `<input>` to tie it to, and there isn't a clear way for screen reader users to read it while they're tabbing through controls or a form. I wonder if we could use `aria-describedby` on the thumbs instead to tie the visible label to the controls?

```
<label id="RangeSlider1Label" for="RangeSlider1" class="Label-Text_SY9aZ">Money spent is between</label>
<div class="DualThumb-Wrapper_q5XRy" id="RangeSlider1">...</div>
```

would become

```
<span id="RangeSlider1Label" class="Label-Text_SY9aZ">Money spent is between</span>
...
<button id="RangeSlider1Lower" class="DualThumb-Thumbs_1FGL0" role="slider" aria-valuemin="0" aria-valuemax="2000" aria-valuenow="900" aria-invalid="false" style="left: 920.728px;" aria-describedby="RangeSlider1Label"></button>
...
<button id="RangeSlider1Upper" class="DualThumb-Thumbs_1FGL0" role="slider" aria-valuemin="0" aria-valuemax="2000" aria-valuenow="1000" aria-invalid="false" style="left: 1023.03px;" aria-describedby="RangeSlider1Label"></button>
```

Reply:
Any chance we could get this tweak in, @tmlayton and @yourpalsonja? It will help prevent automated errors in pa11y once that is up and running again.